docs/React.md

```diff
@@ -100,7 +101,7 @@ Read [a manual](i18n.md).
 
 ## Testing
 
-Testing is done with `Jasmine` and [`Enzyme`](https://github.com/airbnb/enzyme).
+Testing is done with [`jest`](https://jestjs.io/en/) and [`React Testing Library`](https://testing-library.com/docs/react-testing-library/intro).
 
 TODO add an example when references are in master.
```

Comment:
Did you have a nice summary about testing somewhere in chat or Github? It might have some links or other useful bits to add here?

Reply:
Yup, added a bit more now (from https://github.com/Trustroots/trustroots/pull/1147#issuecomment-565741145)